### ISSUE TITLE
suggestion: `9gm` should move tab to the end

### ISSUE THREAD

**Issue by UnkwUsr** [qutebrowser/qutebrowser#7002]
It looks natural to me. Like when we press alt+9 it focus last tab instead of tab number 9.
https://github.com/qutebrowser/qutebrowser/blob/6598914969a38a9d1897c8ff7e254c223c7e8489/qutebrowser/config/configdata.yml#L3622

My temporary solution:

    config.bind("9gm", "tab-move -1")

Maybe it should be added to default config, or is there a way to to do so on source-code level?

Open for discussion.

**Comment by The-Compiler** [comment]
I disagree. Any command which takes a tab number as a count uses `9` to mean the 9th tab. I don't think `gm` should be any different. The only reason that `<Alt-9>` is different is to be consistent with other browser's Alt-*Num* bindings.

**Comment by UnkwUsr** [comment]
Ok, I understand you. Feel free to close issue if there is nothing more to say.
